test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,30 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Server;
+import com.googlecode.jmxtrans.model.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
I added some test (plus minor additional refactoring) on top of your code: https://github.com/jmxtrans/jmxtrans/tree/cloudwatch-writer

For some reason, I can't open a PR to your repo and let you review those. Could you still have a look at https://github.com/jmxtrans/jmxtrans/tree/cloudwatch-writer and tell me if it looks good to you? If it's OK for you, I'll rebase all those changes and merge them (still another PR ...)